docs/source/analyze/node-resources.md

```diff
@@ -124,3 +125,24 @@ spec:
           - pass:
               message: This cluster has a node with enough memory.
 ```
+
+```yaml
+    - nodeResources:
+        checkName: Must have at least 3 nodes with labels beta.kubernetes.io/os=linux and kubernetes.io/rol=database-primary-replica
```

Comment:
Can you describe what these labels mean in plain English?
Also, is there a typo in `rol=`?